opal/runtime/opal_params.c

```diff
@@ -70,7 +69,7 @@ int opal_register_params(void)
 
 #if defined(HAVE_SCHED_YIELD)
     opal_progress_yield_when_idle = false;
-    ret = mca_base_var_register("opal", "opal", "progress", "yield_when_idle",
```

Comment:
The real fix is not to remove it but to make certain we are using it. I would suggest adding
```c
if (0 > ret) {
    return ret;
}
```

Reply:
Will do.